papery/data/sample/themes/default/assets/css/style.css

```
@@ -79,3 +79,6 @@ table tr td {
     border-bottom: 1px solid #e1e1e1;
 }
 
+.emojione, .twemoji, .gemoji {
+  height: 1em;
```

```suggestion
  display: inline-flex;
  height: 1.25em;
  vertical-align: text-top;
```

By applying the emoji style that's actually used on [PyMdown Extensions Documentation](https://facelessuser.github.io/pymdown-extensions/extensions/emoji/), I think it looks better.